extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -802,15 +802,22 @@ export class SqlDatabaseTree {
 			// Reset the dbName list so that it is in sync with the table
 			this._dbNames = this._model._assessmentResults.databaseAssessments.map(da => da.name);
 			this._model._assessmentResults.databaseAssessments.forEach((db) => {
+				let selectable = true;
+				db.issues.forEach((issue) => {
```

Comment:
you should just use the array.find, which will return the first one that matches the criteria, you don't have to go through all the issues.

you can do it inline when you set the enabled property.